src/pages/categories/CategoriesPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const CategoriesPage = () => (
+  <div>categories</div>
```

Comment:
- [ ] **Requirement**
* should display only a button with the text "Check status"